lib/slime/compiler.ex

```diff
@@ -5,27 +5,51 @@ defmodule Slime.Compiler do
 
   alias Slime.Doctype
 
-  alias Slime.Parser.Nodes.{DoctypeNode, EExNode, HTMLCommentNode, HTMLNode, InlineHTMLNode, VerbatimTextNode}
+  alias Slime.Parser.Nodes.{DoctypeNode, EExNode, HEExNode, HTMLCommentNode, HTMLNode, InlineHTMLNode, VerbatimTextNode}
+
+  alias Slime.TemplateSyntaxError
+
+  @eex_delimiters {"#" <> "{", "}"}
```

Comment:
```suggestion
  @eex_delimiters {"\#{", "}"}
```

Any reason not to write this way?